assets/src/features/shared/consts.tsx

```diff
@@ -0,0 +1 @@
+export const MOBILE_BREAKPOINT = 769;
```

Comment:
The width at which the layout changes is a little bit random (768 px is basically a tablet or a reduced window). It's fine for now, but I've addressed the problem of those breakpoints [in this comment](https://www.figma.com/file/lVPaILYT73AJIOsoXNbsXj?node-id=658:17992#382576105)